act-rules-format.bs

```diff
@@ -101,132 +105,150 @@ An ACT Rule MUST have a unique identifier that can be any unique text, such as p
-      </a><ul>
-        <li><strong>Not required</strong> for conformance to WCAG 2.0 and WCAG 2.1 at any level</li>
+    <li>
+      [Technique H37: Using alt attributes on img elements](https://www.w3.org/TR/WCAG20-TECHS/H37.html)
+      <ul>
+        <li>**Not required** for conformance to WCAG 2.0 and WCAG 2.1 at any level</li>
         <li>Outcome mapping: <ul>
-          <li>Failed outcome: not satisfied</li>
-          <li>Passed outcome: satisfied</li>
+          <li>Any failed outcomes: not satisfied</li>
+          <li>All passed outcomes: satisfied</li>
           <li>Inapplicable outcome: satisfied</li>
         </ul></li>
       </ul>
     </li>
-    <li><a href="https://disic.github.io/rgaa_referentiel_en/criteria.html#test-1-1-1">
-      RGAA 3, Test 1.1.1: Does each image have a text alternative
-      </a><ul>
-        <li><strong>Required for conformance</strong> to RGAA 3 level A</li>
+    <li>
+      [RGAA 3, Test 1.1.1: Does each image have a text alternative](https://disic.github.io/rgaa_referentiel_en/criteria.html#test-1-1-1)
+      <ul>
+        <li>**Required for conformance** to RGAA 3 level A</li>
         <li>Outcome mapping: <ul>
-          <li>Failed outcome: not satisfied</li>
-          <li>Passed outcome: satisfied</li>
+          <li>Any failed outcomes: not satisfied</li>
+          <li>All passed outcomes: satisfied</li>
           <li>Inapplicable outcome: satisfied</li>
         </ul></li>
       </ul>
     </li>
   </ul></blockquote>
 </div>
 
-If the `failed` outcome can not be mapped to *not satisfied* for an accessibility requirement, that requirement MUST NOT be included in the accessibility requirements mapping. The optional [Background section](#background) could be used to list accessibility requirements and standards that may be thematically related, but for which the rule is not a failure test.
 
-If the rule does not map to any [=accessibility requirement=], the accessibility requirement mapping will only contain the explainer that it is not required for conformance to the [=accessibility requirements document=]. This is common with atomic rules used in composite rules, where accessibility requirements are only not satisfied with multiple atomic rules have the outcome`failed`.
+### Rules Without Accessibility Requirements ### {#rules-without-accessibility-requirements}
+
+If the rule does not map to any [=accessibility requirement=], the accessibility requirement mapping will only contain the explainer that it is not required for conformance to the [=accessibility requirements document=]. This is common with atomic rules used in composite rules.
 
 <div class=example>
-  <p><strong>Example:</strong> An ACT Rule that tests a best practice not included in any accessibility standard:</p>
+  <p>**Example:** An ACT Rule for WCAG 2.1 that tests if landmarks are used to satisfy [WCAG 2.1, success criterion 2.4.1 bypass blocks](https://www.w3.org/TR/WCAG21/#bypass-blocks):</p>
```

Comment:
This example is also a Technique (ARIA 11). How is this example different from the first example above for Technique 37? I thought a best practice might be a good example here. Left alignment of paragraph text (or similar).